src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/noslow/modes/slowness/NoSlowSlowness.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.movement.noslow.modes.slowness
+
+import net.ccbluex.liquidbounce.config.types.ToggleableConfigurable
+import net.ccbluex.liquidbounce.event.tickHandler
+import net.ccbluex.liquidbounce.features.module.modules.movement.noslow.ModuleNoSlow
+import net.minecraft.entity.attribute.EntityAttributeModifier
+import net.minecraft.entity.attribute.EntityAttributes
+import net.minecraft.entity.effect.StatusEffects
+import net.minecraft.util.Identifier
+
+internal object NoSlowSlowness : ToggleableConfigurable(ModuleNoSlow, "Slowness", true) {
+    val multiplier by float("PerLevelMultiplier", 0f, 0f..0.15f)
+
+    @Suppress("unused")
+    val tickHandler = tickHandler { setSlownessMultiplier(multiplier) }
+
+    private fun setSlownessMultiplier(multiplier: Float) {
+        val slowness = player.activeStatusEffects[StatusEffects.SLOWNESS]?.amplifier ?: return
+        player.attributes.getCustomInstance(EntityAttributes.GENERIC_MOVEMENT_SPEED)?.updateModifier(
+            EntityAttributeModifier(
+                Identifier.of("effect.slowness"),
+                -multiplier * (slowness + 1.0),
+                EntityAttributeModifier.Operation.ADD_MULTIPLIED_TOTAL
+            )
+        )
```

Comment:
I don't like the idea of editing the effect like that. It should really hook it where it's applied, I guess?

Reply:
but then we can't modify pre existing effects

Reply:
No, I mean where it applies the effect. Not where the effect is applied.

Reply:
If you mean this, it does nothing
![image](https://github.com/user-attachments/assets/c31c042e-994d-4ee8-9399-ef7c6f637767)

Reply:
Yes, then fix that?

Reply:
the reason this does nothing is because the attribute is applied by the server and not the client, we would have to modify the attribute when we receive the attribute packet, which as I said before, won't let us disable this, or modify the multiplier after the effect is applied

Reply:
But your implementation has the same flaw, no?

Reply:
no, this implementation allows that